rotkehlchen/chain/evm/tokens.py

```diff
@@ -127,9 +133,9 @@ def __init__(
     def get_token_balances(
             self,
             address: ChecksumEvmAddress,
-            tokens: list[EvmToken],
+            tokens: list[EvmTokenDetectionData],
             call_order: Sequence[WeightedNode] | None,
-    ) -> dict[EvmToken, FVal]:
+    ) -> dict[Asset, FVal]:
```

Comment:
Can you specify in the docstring why you return an `Asset` instead of an `EvmToken`? I know why -- this is why you made this PR, but should also be explained in the docstring.